RELEASE.md

```diff
@@ -19,8 +19,13 @@ The commands below assume a production application named `dbt-hubcap`. Replace w
     - Job: `python3 hubcap.py`
     - Schedule: Every hour at :00
     - Dyno size: Hobby / Standard-1X
-1. Configure the `CONFIG` environment variable: [Settings > Config Vars > Reveal Config Vars](https://dashboard.heroku.com/apps/dbt-hubcap/settings)
+1. Configure environment variables: [Settings > Config Vars > Reveal Config Vars](https://dashboard.heroku.com/apps/dbt-hubcap/settings)
     - `CONFIG`: copy format from `config.example.json` and adjust values as needed
+    - `GIT_EXEC_PATH`: `/app/.apt/usr/lib/git-core`
```

Comment:
Note: setting the `GIT_EXEC_PATH` environment variable won't be necessary anymore if https://github.com/heroku/heroku-buildpack-apt/issues/137 is implemented.